src/psycopmlutils/loaders/raw/load_coercion.py

```diff
@@ -0,0 +1,113 @@
+        n: Optional[int] = None,
+    ) -> pd.DataFrame:
+        """Load coarcion data.
+
+        Args:
+            coercion_type (str): Type of coercion, e.g. 'tvangsindlæggelse', 'bæltefiksering'. # noqa: DAR102
+            reason_for_coercion (str): Reason for coercion, e.g. 'farlighed'. # noqa: DAR102
+            n: Number of rows to return. Defaults to None.
+
+        Returns:
+            pd.DataFrame
+        """
+        view = "[FOR_tvang_alt_hele_kohorten_inkl_2021]"
+
+        sql = f"SELECT dw_ek_borger, datotid_start_sei, varighed FROM [fct].{view}"
+
+        if coercion_type and reason_for_coercion is None:
+
+            sql += f"WHERE typetekst_sei = '{coercion_type}'"
+
+        if coercion_type is None and reason_for_coercion:
+
+            sql += f"WHERE begrundtekst_sei = '{reason_for_coercion}'"
+
+        if coercion_type and reason_for_coercion:
+
+            sql += f"WHERE typetekst_sei = '{coercion_type}' AND begrundtekst_sei = '{reason_for_coercion}'"
+
+        df = sql_load(sql, database="USR_PS_FORSK", chunksize=None, n=n)
+
+        df.rename(
+            columns={"datotid_start_sei": "timestamp", "varighed": "value"},
+            inplace=True,
+        )
+
+        # msg.good(f"Loaded {print_str}")
+        return df.reset_index(drop=True)
+
+    def _aggregate_coercion(
+        coercion_types_lists: list,
+        subset_by: Optional[int] = "both",
+        n: Optional[int] = None,
+    ) -> pd.DataFrame:
+        """Aggregate multiple types of coercion with multiple reasons into one
+        data frame column.
+
+        Args:
+            coercion_types_lists (list): List of lists each containing a string with coercion_type and a string with reason_for_coercion # noqa: DAR102
+            subset_by (str): String indicating whether data is being subset based on coercion type, reason or both
+            n (int, optional): Number of rows to return. Defaults to None.
+
+        Returns:
+            pd.DataFrame
+        """
+        subset_choices = ["type", "reason", "both"]
+
+        if subset_by not in subset_choices:
+            raise ValueError(
+                "Invalid subset_by argument. Expected one of: %s" % subset_choices,
+            )
+
+        if subset_by == "both":
+            dfs = [
+                LoadCoercion.coercion(
+                    coercion_type=id[0],
+                    reason_for_coercion=id[1],
+                    n=n,
+                )
+                for id in coercion_types_lists
+            ]
+
+        if subset_by == "type":
+            dfs = [
+                LoadCoercion.coercion(coercion_type=id[0], n=n)
+                for id in coercion_types_lists
+            ]
+
+        if subset_by == "reason":
+            dfs = [
+                LoadCoercion.coercion(reason_for_coercion=id[0], n=n)
+                for id in coercion_types_lists
+            ]
+
+        return pd.concat(dfs, axis=0).reset_index(drop=True)
```

Comment:
A list of dicts could also simplify this. E.g. your first if statement becomes `if "coercion_type" in d` – saving us an argument in the function!